src/Slices/ResourceV2/UI/resourceDetailsTab.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from "react";
+import { Flex, FlexItem } from "@patternfly/react-core";
+import styled from "styled-components";
+import { Query, RemoteData } from "@/Core";
+import { useUrlStateWithString } from "@/Data";
+import { TabKey, Tabs } from "@/Slices/ResourceDetails/UI/Tabs";
+import { DependencyContext, words } from "@/UI";
+import {
+  Description,
+  PageContainer,
+  ResourceStatusLabel,
+} from "@/UI/Components";
+
+interface ResourceTabsProps {
+  resourceId: string;
+}
+
+export const ResourceDetailsTab: React.FunctionComponent<ResourceTabsProps> = ({
```

Comment:
There is also the `React.FC` shorthand instead of `React.FunctionComponent`, both are fine